Tutorials/index.md

```diff
@@ -1,253 +1,32 @@
 ---
 toc: true
-redirect_from:
-  - /projects/sel4-tutorials.html
-  - /projects/sel4-tutorials/
-layout: project
-project: sel4-tutorials
+layout: overview
 SPDX-License-Identifier: CC-BY-SA-4.0
-SPDX-FileCopyrightText: 2020 seL4 Project a Series of LF Projects, LLC.
+SPDX-FileCopyrightText: 2024 seL4 Project a Series of LF Projects, LLC.
 ---
-
-# Tutorials
-
-{% assign tutorials = site.pages | where_exp: 'page', 'page.tutorial' | sort: 'tutorial-order' %}
+# Tutorials about seL4
 
 We have developed a series of tutorials to introduce seL4 and
 developing systems on seL4.
 
-## How to use the tutorials
-
-Depending on your goals and what you want to do with seL4, we suggest
-different paths to follow through the tutorial material.  Choose the
-most relevant category below and follow the tutorials in the suggested
-order.
```

Comment:
I do think that section and set of paths was useful. We should not remove that completely, just recommend a default path (which the site navigation basically suggests now). There is a lot of useful content overview in here that we could use in the overview page.

Reply:
Much of this page has been put into the new Overview and Pathways pages, which describe the tutorials and alternate pathways respectively.